packages/react-components/react-swatch-picker-preview/src/components/SwatchPicker/useSwatchPickerStyles.styles.ts

```diff
@@ -20,20 +18,25 @@ const { gridGap } = swatchPickerCSSVars;
 const useStyles = makeStyles({
   root: {
     ...shorthands.padding(tokens.spacingHorizontalNone, tokens.spacingVerticalNone),
+    display: 'flex',
   },
   row: {
-    display: 'flex',
     flexDirection: 'row',
     columnGap: `var(${gridGap})`,
   },
+  grid: {
+    flexDirection: 'column',
+    rowGap: `var(${gridGap})`,
```

Comment:
Should it be `...shorthands.gap()` instead?

Reply:
in case with grid it's not important to have `gap` because row component has it's own gap for swatches

Reply:
in case if we need responsive layout I'll change it to `gap`